help/md/readTrace.md

```diff
@@ -1,16 +1,31 @@
 ## name
 readTrace
 ## title
-readTrace
-## description
 Read an MCMC log file.
+## description
+Reads parameter values from a log file,  usually produced as the output of an MCMC or MCMCMC run.
 ## details
+
 Read an MCMC log file with field delimited by `separator`.
 Then drop the first `burnin` iterations if `burnin` is an integer,
 or the fraction `burnin` of iterations if `burnin` if a Real number.
-Then we keep every `n`th entry if the `thinning` is `n`.
+Then we keep every $n$th entry if the `thinning` is $n$.
```

Comment:
Sadly we don't handle LaTeX, so I think the `$` should show up in the output if you type `?readTrace`